client/src/main/java/org/evosuite/TestSuiteGenerator.java

```diff
@@ -1,19 +1,19 @@
 /**
  * Copyright (C) 2010-2018 Gordon Fraser, Andrea Arcuri and EvoSuite
  * contributors
- *
+ * <p>
```

Comment:
Is this on purpose? There is a Maven plugin that updates all headers automatically, so if you want to change the copyright header we should use that and make a single commit for the copyright updates

Reply:
The changes were not on purpose, i will remove this lines